src/main/java/org/frc5687/powerup/robot/Robot.java

```diff
@@ -149,6 +152,19 @@ public void testPeriodic() {
     public void updateDashboard() {
         pdp.updateDashboard();
         _autoChooser.updateDashboard();
+        SmartDashboard.putBoolean("Competition bot", _isCompetitionBot);
```

Comment:
`_isCompetitionBot` is only set during `robotInit()`, so I don't see the point of having a SmartDashboard call 50 times per second when it's essentially a constant after the first 50th of a second.